src/parser/parser.yy

```diff
@@ -1638,6 +1640,9 @@ show_sentence
     }
     | KW_SHOW KW_COLLATION {
         $$ = new ShowSentence(ShowSentence::ShowType::kShowCollation);
+    }
+	| KW_SHOW KW_PLUGINS {
```

Comment:
Please alignment the space :)

Reply:
Good catch，it may cause by mixing tabs or spaces.